tests/components/version-list-row-test.js

```diff
@@ -44,4 +44,46 @@ module('Component | VersionList::Row', function (hooks) {
     assert.dom('[data-test-release-track]').hasText('?');
     assert.dom('[data-test-release-track-link]').hasText(version);
   });
+
+  test('pluralize "feature" only when appropriate', async function (assert) {
+    let crate = this.server.create('crate', { name: 'foo' });
+    this.server.create('version', {
+      crate,
+      num: '0.1.0',
+      features: {},
+      created_at: Date.now(),
+      updated_at: Date.now(),
+    });
+    this.server.create('version', {
+      crate,
+      num: '0.2.0',
+      features: { one: [] },
+      created_at: Date.now(),
+      updated_at: Date.now(),
+    });
+    this.server.create('version', {
+      crate,
+      num: '0.3.0',
+      features: { one: [], two: [] },
+      created_at: Date.now(),
+      updated_at: Date.now(),
+    });
+
+    let store = this.owner.lookup('service:store');
+    let crateRecord = await store.findRecord('crate', crate.name);
+    let versions = (await crateRecord.versions).toArray();
+    this.firstVersion = versions[0];
+    this.secondVersion = versions[1];
+    this.thirdVersion = versions[2];
+
+    await render(hbs`<VersionList::Row @version={{this.firstVersion}} />`);
+    assert.dom('[data-test-feature-list]').hasText('0 Features');
+
+    await render(hbs`<VersionList::Row @version={{this.secondVersion}} />`);
+    assert.dom('[data-test-feature-list]').hasText('1 Feature');
+    assert.dom('[data-test-feature-list]').doesNotIncludeText('Features');
```

Comment:
```suggestion
```

`hasText()` compares via `==` and since we know that it is `1 Feature` we can be sure that it can't include `Features` at the same time 😉